src/wishes/wishes.controller.ts

```diff
@@ -44,24 +51,28 @@ export class WishesController {
     return this.wishesService.updateWish(id, updateWish);
   }
 
+  @UseGuards(AuthGuard)
   @Delete(':id')
   deleteWish(@Param('id') id: string) {
     return this.wishesService.deleteWish(id);
   }
 
+  @UseGuards(AuthGuard)
   @Post(':id/categories/:category')
   createCategoryForWish(
     @Param('id') id: string,
     @Param('category') category: string,
   ): Promise<void> {
     return this.wishesService.createCategoryForWish(id, category);
   }
-  
+
+  @UseGuards(AuthGuard)
   @Get('categories/:id')
```

Comment:
endpoints for categories should be extracted into another Controller CategoriesController